tasks/89_symmertic-difference/symmertic-difference.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-len */
+const symDiff = (a, b) => {
+  const result = [];
+
+  a = [...new Set( a.sort() )];
+
+  b = [...new Set( b.sort() )];
+
+  a.map(( element ) => {
+    if ( b.indexOf( element ) === -1 ) {
```

Comment:
result if indexOf better save to variable

Reply:
Done